tuneme/templates/new/registration/login.html

```diff
@@ -2,28 +2,39 @@
 {% load i18n %}
 
 {% block content %}
-<div class="heading heading--auth">{% trans "Log in" %}</div>
-
-
-<div class="auth-tools">
-
-	<a href="{% url 'molo.profiles:user_register' %}">{% trans "Don't have an account? Register" %}</a>
-
-	<form method="post" action="">
-		{% csrf_token %}
+<h1 class="heading heading--profiles">{% trans "Log in" %}</h1>
+<div class="call-to-action call-to-action--profile">
+    <h5 class="call-to-action__nav-item call-to-action__nav-item--challenge">
+    {% trans "Don't have an account yet?" %}<br/>
+    <a href="{% url 'molo.profiles:user_register' %}" class="call-to-action__link">
+    {% trans "Click here" %}</a> {% trans "to join today!" %}</h5>
```

Comment:
Mobile phone user's don't click; they tap. Let's reword this sentence to:

`Don't have an account yet? _Join!_`